CSS/index.css

```diff
@@ -0,0 +1,99 @@
+body {
+    margin: 0;
+    background-color: rgb(50, 50, 50);
+    color: white;
+    font-family: "Playfair Display", Verdana, Geneva, Tahoma, sans-serif;
+}
+
+h1 {
+    text-align: center;
+    color: #eb8d46;
+}
+
+h2 {
+    color: #eb8d46;
+}
+
+main {
+    padding: 120px 20px;
+}
+
+a {
+    color: #8c98bc;
+    font-style: italic;
+}
+
+.boldFont {
+    font-weight: bolder;
+}
+
+nav {
+    display: flex;
+    gap: 30px;
+    justify-content: flex-end;
+    margin: 30px;
+}
+
+.navItem {
+    color: #8c98bc;
+    background-color: white;
+    padding: 10px;
+    font-style: normal;
+    font-size: 20px;
+    text-decoration: none;
+    font-weight: bold;
+}
+
+header {
+    background-color: black;
+}
+
+.sticky {
```

Comment:
To make it not scroll over the sections, for the section headers you should add something like: 

#About{
    scroll-margin-top: 5em;
}

That should help with the scrolling issues.

Reply:
Updated